cirq-aqt/cirq_aqt/aqt_device.py

```diff
@@ -36,11 +37,18 @@
 gate_dict = {'X': cirq.X, 'Y': cirq.Y, 'Z': cirq.Z, 'MS': cirq.XX, 'R': cirq.PhasedXPowGate}
 
 
+class OperationString(Enum):
```

Comment:
We should add a short docstring for this enum.

Reply:
Thanks - fixed.